### DISCUSSION TITLE
I can't connect to Copilot when using VPN

### DISCUSSION THREAD

**Discussion by zhouyang-bigdata** [community/community#64573] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

When using a VPN, I cannot connect to Copilot. I tried many times on Visual Studio and Pycharm IDE.
Before 4 days, it was connectable.
![Snipaste_2023-08-22_12-10-24](https://github.com/community/community/assets/47346770/5b5dcc9f-1ae6-4cbf-86c7-7434a41ae45d)

**Comment by donno2048** [comment, marked as answer]
Try uninstalling the extension, closing VSCode, reopening VSCode, and reinstalling the extension all with your VPN on..? But before you do just make sure your extension is updated to the current version.

**Reply by zhouyang-bigdata** [reply]
thanks. You may be right. I tried several times, I even re-logged VSCODE IDE with github account. Eventually I was able to connect to copilot.
But the new problem is that copilot does not show the auto-completion feature. Maybe I need to tinker a little more.

**Comment by arafajs** [comment]
That happens when you disable Vpn for once and try to use the copilot. Try to log in to another account for GitHub and Don't forget to turn on VPN. Otherwise, it would always disconnect. (Due to Logging in from different Locations)

**Reply by zhouyang-bigdata** [reply]
thanks. You may be right.

**Comment by 0xtane** [comment]
Disable the extension > Reload VSCode > Enable > Done
